openvdb/points/AttributeArray.h

```diff
@@ -256,17 +256,17 @@ class OPENVDB_API AttributeArray
     template<typename ValueType>
     bool hasValueType() const { return this->type().first == typeNameAsString<ValueType>(); }
 
-    /// Set value at given index @a n from @a sourceIndex of another @a sourceArray
-    /// @note deprecated from ABI=6, use copyValues() with source-target index pairs
+    /// @brief Set value at given index @a n from @a sourceIndex of another @a sourceArray.
```

Comment:
Tag short and long descriptions with `@brief` and `@details`, respectively.

Reply:
> @danrbailey I don't seem to be able to add you as a reviewer. Can you look over these fixes?

These all look good to me, thanks @pcucka. I think all of these changes but particularly the doxygen ones highlight a need to extend our CI to automate these checks.

Reply:
Agreed, and ideally we should enable warnings as errors.